contracts/RewardPool.sol

```diff
@@ -393,6 +557,12 @@ contract RewardPool is AccessControl, ReentrancyGuard {
     reserveAddress = _reserveAddress;
   }
 
+  function _setRewardFunctionAddress(address _rewardFunctionAddress) internal {
+    if (_rewardFunctionAddress == address(0))
+      revert InvalidRewardFunctionAddress();
+    rewardFunctionAddress = _rewardFunctionAddress;
+  }
```

Comment:
Do we need this function? I only see it called once from `setRewardFunctionAddress`.